content/reports/240315-contributors.md

```diff
@@ -0,0 +1,33 @@
+---
+type: contributors
+description: |
+  Hive version for TDP stack 2.0
+---
+
+# Notes from March 15th, 2024
+
+Hive version for TDP stack 2.0
```

Comment:
We are currently looking at Hive 4 but we have concern about integration with Ranger (Hive 4 not compatible with Ranger for now). For Hive 3, it is not working with Hadoop 3.3, Bigtop project patch Hive in order to work with Hadoop 3.3.
For now the work begin with integrating Hive 3 with Hadoop 3.3.